std-bits/table/src/main/java/org/enso/table/expressions/ExpressionVisitorImpl.java

```diff
@@ -67,43 +67,16 @@ public interface MethodInterface {
 
   public record MethodResolver(
       Value module, Value type, boolean isStaticMethod, Function<Object, Value> makeTypedColumn) {
-    public MethodResolver(
-        String moduleName,
-        String typeName,
-        boolean isStaticMethod,
-        Function<Object, Value> makeTypedColumn) {
-      this(
-          Context.getCurrent().getBindings("enso").invokeMember("get_module", moduleName),
```

Comment:
- I've discovered this line while working on #13851
- One **cannot use** `Context.getCurrent()` in the _dual JVM_ as there are **two contexts**
- rather than that passing in real Enso objects